.pre-commit-config.yaml

```diff
@@ -1,28 +1,28 @@
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v1.2.3
+    rev: v2.4.0
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
```

Comment:
I think, need to add exclude for `.bumpversion.cfg`, because every time bumpversion will add 2 empty lines at the end of file after version bumping

Reply:
Good point! I fixed it.